cacheops/invalidation.py

```diff
@@ -14,10 +14,15 @@
 from .signals import cache_invalidated
 from .transaction import queue_when_in_transaction
 from .local_cache import RequestLocalCacheObj
+import logging
+from datetime import datetime
+import pytz
 
 __all__ = ('invalidate_obj', 'invalidate_model', 'invalidate_all', 'no_invalidation')
 
 
+redis_logger = logging.getLogger('redis.log')
```

Comment:
i think this should be configurable during setup of cacheops other wise we have to create a logger no matter if we need or not